clarifai.go

```diff
@@ -175,9 +175,9 @@ func getRequestBody(input string) (*bytes.Buffer, error) {
 }
 
 // Get image tag array from Clarifai's API response
-func getImageTagsArray(jsonStr string) ([]ClarifyTag, error) {
+func getImageTagsArray(jsonStr string) ([]ImageTag, error) {
```

Comment:
Yes everything is always accessible in the same package. But only functions, struct fields, methods, etc. that start with a capital letter will be accessible outside the package. I guess this will not be a problem for us as we'll likely only have 1 package and our project would be never used as a dependency to something else